README.md

```diff
@@ -14,32 +14,58 @@
 
 ## Installation
 
+In the project navigator select a target, navigate to `Build Settings` and set `Enable Bitcode` to `No`.
+Add ` -ObjC` to `Other Linker Flags`
+
+Add  build phase with name `Crashlytics` with following content
+`"$PROJECT_DIR/Carthage/Build/iOS/FirebaseBroad.framework/run"`
+and pass DSYM and info.plist as input files of the scrypt
+```
+${DWARF_DSYM_FOLDER_PATH}/${DWARF_DSYM_FILE_NAME}/Contents/Resources/DWARF/${TARGET_NAME}
+$(SRCROOT)/$(BUILT_PRODUCTS_DIR)/$(INFOPLIST_PATH)
+```
+
 #### Carthage
 Create a `Cartfile` that lists the framework and run `carthage update`. Follow the [instructions](https://github.com/Carthage/Carthage#adding-frameworks-to-an-application) to add the framework to your project.
 
 ```
-github "ntunin/FirebaseBroad"
+github "rosberry/ButterBroad"
+github "rosberry/CrashlyticsBroad"
 ```
 Add following frameworks from Carhage/Build/iOS folder:
 - AnyCodable.framework
 - Butterbroad.framework
 - FirebaseBroad.framework
-- Firebase.framework
+- FirebaseAnalytics.framework
 
 ### Manually
 
 Drag `Sources` folder from [last release](https://github.com/rosberry/FirebaseBroad/releases) into your project.
 
 ## Usage
 
+In the AppDelegate
+
+```swift
+import Firebase
+
+
+func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: LaunchOptions?) -> Bool {
+    ...
+    FirebaseApp.configure()
+    Butter.activationHandler?()
+    ...
+}
+```
+
 #### Creating a broad
 
 ```swift
 import ButterBroad
 import FirebaseBroad
 
 extension Butter {
-    static let firebase: FirebaseBroad = .init()
+static let firebase: FirebaseBroad = .init(Firebase.Analytics.self)
```

Comment:
Wrong indentation and usage of `Firebase.Analytics.self` is strange. Can't we use it inside of the broad?

Reply:
Yes, strange, but it is necessary to make Firebase Analytics instance initialized in the context that FirebaseBroad should not  have direct references to firebase